src/controllers/expenseController.js

```diff
@@ -0,0 +1,94 @@
+const expensesService = require('../services/expensesService');
+const userService = require('../services/userService');
+
+const get = (req, res) => {
+  const { userId, categories, from, to } = req.query;
+  const filteredExpenses = expensesService.getAllExpenses(
+    userId,
+    categories,
+    from,
+    to,
+  );
+
+  res.status(200).send(filteredExpenses);
```

Comment:
It's a good practice to always return a JSON object from your API endpoints, even if it's an array. This allows you to add additional metadata if necessary and makes your API more consistent. Consider wrapping `filteredExpenses` in an object before sending it.